src/dscim/preprocessing/input_damages.py

```diff
@@ -863,3 +874,50 @@ def coastal_inputs(
             consolidated=True,
             mode="w",
         )
+
+
+def validate_damages(sector, path):
+    inputs = xr.open_zarr(path)
+    inputs.close()
+
+    # No repeated batch labels
+    batches_expected = np.sort(["batch" + str(i) for i in np.arange(0, 15)])
+    batches_actual = np.sort(inputs.batch.values)
+    assert np.array_equal(
+        batches_expected, batches_actual
+    ), f"Batches in the {sector} input damages zarr are not 0-14."
+
+    # Input damages have rcp 4.5 and rcp 8.5
+    if "coastal" not in sector:
+        rcps_expected = np.sort(["rcp" + str(i) for i in [45, 85]])
+        rcps_actual = np.sort(inputs.rcp.values)
+        assert np.array_equal(
+            rcps_expected, rcps_actual
+        ), f"RCPs in the {sector} input damages zarr are not rcp45 and rcp85."
+
+    # max batches and no repeated batches
+    regions = inputs.dims["region"]
+    ssps = inputs.dims["ssp"]
+    if "coastal" in sector:
+        dims = ["ssp", "model", "slr", "batch", "year", "region"]
+        chunk_sizes = [1, 1, 1, 15, 10, regions]
+        total_sizes = [ssps, 2, 10, 15, 90, regions]
+    else:
+        dims = ["ssp", "rcp", "model", "gcm", "batch", "year", "region"]
+        chunk_sizes = [1, 1, 1, 1, 15, 10, regions]
+        total_sizes = [ssps, 2, 2, 33, 15, 90, regions]
+
+    chunk_len = np.arange(0, len(chunk_sizes))
+    chunks = [
+        (chunk_sizes[i],) * int(total_sizes[i] / chunk_sizes[i]) for i in chunk_len
+    ]
+    dims_expected = dict(zip(dims, total_sizes))
+    chunks_expected = dict(zip(dims, chunks))
+
+    assert dims_expected == dict(inputs.dims)
+    for i in list(inputs.keys()):
```

Comment:
Might be able to just have this as `inputs.keys()` without casting it to a list?